src/systems/ec/specs/components/inpage-navigation/docs/usage.md

```diff
@@ -1 +1,15 @@
-Do's and don'ts
+The in-page navigation is a vertical menu containing (anchored) links to H2-headings on long content pages. It appears at the left side of a page. It gives the user an overview of the page contents and allows them to jump directly to content that is lower on the page without having to scroll through the rest of the page.  Most content pages on ec.europa.eu make use of the in-page navigation component.
```

Comment:
The documentation says:

> It appears at the left side of a page.

The ECL website shows one on the right side. Kind of confusing, isn't it?

Reply:
We put this on hold.